beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -1160,6 +1160,40 @@ impl<E: EthSpec> Network<E> {
         peer_id: PeerId,
         response: Response<E>,
     ) -> Option<NetworkEvent<E>> {
+        match &response {
+            Response::Status(_) => {
+                metrics::inc_counter_vec(&metrics::TOTAL_RPC_RESPONSES_RECEIVED, &["status"])
```

Comment:
You can derive `IntoStaticStr` for `Response` and just do a single
```rust
metrics::inc_counter_vec(&metrics::TOTAL_RPC_RESPONSES_RECEIVED, &[response.into()])
```

Reply:
done